### PR TITLE
[hermes] Add NNCC_LIBRARY_NO_PIC option to CMakeLists.txt

### PR DESCRIPTION
**Pull request by SlavikMIPT**
This commit adds NNCC_LIBRARY_NO_PIC option to have an opportunity to disable POSITION_INDEPENDENT_CODE property.

ONE-DCO-1.0-Signed-off-by: Vyacheslav Bazhenov slavikmipt@gmail.com

### CODE REVIEW

File: compiler/hermes/CMakeLists.txt [hunk 1]
```diff
@@ -3,7 +3,11 @@ file(GLOB_RECURSE TESTS "src/*.test.cpp")
 list(REMOVE_ITEM SOURCES ${TESTS})
 
 add_library(hermes STATIC ${SOURCES})
-set_target_properties(hermes PROPERTIES POSITION_INDEPENDENT_CODE ON)
+
```
Comment (m-bronnikov) [inline compiler/hermes/CMakeLists.txt:6]: In other PR's you don't adds empty line before `if` statement. Redundant?

Comment (binarman) [reply compiler/hermes/CMakeLists.txt]: Removed =)

**Review by m-bronnikov — APPROVED** [review]
LGTM

**Comment by binarman** [comment]
CMake variable is renamed to comply with #8071

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by binarman — APPROVED** [review]

**Review by m-bronnikov — APPROVED** [review]
LGTM

**Pull request merged**